app/__init__.py

```diff
@@ -7,35 +7,40 @@
 import shutil
 import logging
 import subprocess
-from response_messages import errors, response
+from app.response_messages import response, errors
 from logging.handlers import RotatingFileHandler
-from flask import Flask, jsonify, request, Response
+from flask import Flask, jsonify, request, Response, abort, make_response
+from werkzeug.security import generate_password_hash, check_password_hash
 from flask_swagger_ui import get_swaggerui_blueprint
-from utils import get_connection, create_fetch_cmd, create_cmd_workspace,\
-    create_cmd_up_pinfile, check_workspace_empty
-
+from functools import wraps
+from app.utils import get_connection, create_fetch_cmd, create_cmd_workspace,\
+    create_cmd_up_pinfile, check_workspace_empty, get_connection_users, \
+    create_admin_user
 
 app = Flask(__name__)
 
 APP_DIR = os.path.dirname(os.path.realpath(__file__))
 
-# Reading directory path from config.yml file
 try:
     with open(APP_DIR + '/config.yml', 'r') as f:
         config = yaml.load(f)
-except Exception as e:
+except Exception as x:
```

Comment:
@mansikulkarni96 why x instead of e?

Reply:
changed it because it was being used globally and shadowing other usages, I could use some other variable related to exception.